eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -121,16 +121,40 @@
     <!-- Known issue in autorest where it will generate a variable with a value of null then use it in a not null check.
     https://github.com/Azure/autorest.java/issues/328 -->
     <Match>
-        <Class name="~com\.microsoft\.azure\.batch\.protocol\.implementation\.[\w]+Impl"/>
-        <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE"/>
+      <Class name="~com\.microsoft\.azure\.batch\.protocol\.implementation\.[\w]+Impl"/>
+      <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE"/>
     </Match>
 
     <Match>
-        <Class name="com.microsoft.azure.batch.CreateTasksErrorException"/>
-        <Or>
-            <Field name="failureTaskList"/>
-            <Field name="pendingTaskList"/>
-        </Or>
-        <Bug pattern="SE_BAD_FIELD"/>
+      <Class name="com.microsoft.azure.batch.CreateTasksErrorException"/>
+      <Or>
+        <Field name="failureTaskList"/>
+        <Field name="pendingTaskList"/>
+      </Or>
+      <Bug pattern="SE_BAD_FIELD"/>
+    </Match>
+
+    <!-- Methods in Common package return null instead of meaningful instances when resourses do not exist. Will address the bugs in Track Two.
+    -->
+    <Match>
+      <Class name="~com\.azure\.common\.implementation\.(Base64Url|serializer\.HttpResponseDecodeData|util\.CloneUtils)"/>
+      <Bug pattern="PZLA_PREFER_ZERO_LENGTH_ARRAYS"/>
+    </Match>
+    <Match>
+      <Class name="com.azure.common.implementation.Base64Url"/>
+      <Bug pattern="NP_TOSTRING_COULD_RETURN_NULL"/>
+    </Match>
```

Comment:
You should not suppress this - the common module **is** track two.

Reply:
What should we do if not suppress? The bug is complaining about return null when string is null. The method is public, so I am afraid it will break the call if change the logic.

Reply:
This code is not yet being used by anyone, so it should be fixed appropriately.

Reply:
Got your point. Will do

Reply:
Done.